magento/pwa-studio
# [Docs] Add a static route docs - Snippet correction

#2294  Description

Update to a snippet in "Import new App component" section so opening and closing quote match. Currently, an incorrect character used to close the statement causes an error when trying to follow the tutorial.

## Related Issue






## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps
1. Copy and paste the current snippet into an IDE to see that the opening and closing characters do not match. 

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have updated the documentation accordingly, if necessary.
